src/bin/miri.rs

```diff
@@ -227,10 +227,6 @@ fn main() {
                 "-Zmiri-symbolic-alignment-check" => {
                     miri_config.check_alignment = miri::AlignmentCheck::Symbolic;
                 }
-                "-Zmiri-disable-isolation" => {
```

Comment:
Please don't remove this flag. It is very widely used, so this would break tons of projects' CI.

Reply:
I misunderstood your previous comment about deleting the flag. Sorry about that. I restored the flag.

Reply:
Oh I see what happened. Sorry for not being more clear.